### PR TITLE
pycodestyle recolor_season_variants.py

### PR DESCRIPTION
**Pull request by int-ua**
#### Summary
Infrastructure "pycodestyle recolor_season_variants.py"

#### Content of the change
pycodestyle and other edits and comments

#### Testing
TODO

### CODE REVIEW

**Comment by Fris0uman** [comment]
This does look way cleaner 👍

**Comment by int-ua** [comment]
@secretstamos I understand that it can be annoying since you seem to like camel case and since I've edited almost all of it. Sorry, but Python generally has differing conventions. Are you ok with these changes? Some of them are a bit of personal preference. Can you please comment the `TODO: describe` parts?

**Comment by secretstamos** [comment]
> @secretstamos I understand that it can be annoying since you seem to like camel case and since I've edited almost all of it. Sorry, but Python generally has differing conventions. Are you ok with these changes? Some of them are a bit of personal preference. Can you please comment the `TODO: describe` parts?

Feel free to make any changes you'd like

**Comment by int-ua** [comment]
It's not immediately clear what exactly these blocks do without spending more time which I decided to postpone until I have more free time.

**Comment by secretstamos** [comment]
> It's not immediately clear what exactly these blocks do without spending more time which I decided to postpone until I have more free time.

@int-ua I'll comment on what they do. Thanks!

File: tools/recolor_season_variants.py [hunk 1]
```diff
@@ -1,77 +1,124 @@
...
+    """
+    replacement_colors = SEASON_PREFIXES[season]
+    img = Image.open(season_file).convert('RGB')
+    data = numpy.array(img)
+    current_color = 0
+
+    for color in DEFAULT_COLORS:
+        # TODO: describe
+        data[(data == color).all(axis=-1)] = replacement_colors[current_color]
+        current_color = current_color + 1
```
Comment (secretstamos) [inline tools/recolor_season_variants.py:55]: This block is a little hard to explain, but here goes. 
We're iterating through each default (spring) color. We've used numpy to turn the image into an array (var data). As we iterate through each spring color we're also iterating through the replacement colors array. We're basically mass replacing the default colors, starting with the lightest and moving to the darkest.
Afterwards we use numpy to reconstruct the image. This method (replace_colors) is run for each alternative season (summer, fall and winter).

**Pull request merged**